lib/src/widget/pin_widget.dart

```diff
@@ -303,14 +313,14 @@ class PinInputTextFormField extends FormField<String> {
     bool enabled = true,
     FormFieldSetter<String> onSaved,
     FormFieldValidator<String> validator,
-    bool autovalidate = false,
+    AutovalidateMode autovalidateMode = AutovalidateMode.disabled,
```

Comment:
it would be better to keep it coz there are some applications use this library with outdated flutter version

Reply:
we can keep it and add `@Deprecated`, and use your new field

Reply:
// FIXME: remove it later
// ignore: deprecated_member_use
would help on this issue

Reply:
No problem, I can add this back in 👍